godot-core/src/builtin/array.rs

```diff
@@ -675,6 +675,26 @@ impl<T: GodotType> fmt::Debug for Array<T> {
     }
 }
 
+impl<T: GodotType + fmt::Display> fmt::Display for Array<T> {
+    /// Formats `Array` to match Godot's string representation
+    ///
+    /// Example:
+    /// ```no_run
+    /// # use godot::prelude::*;
+    /// let array = array![1,2,3,4];
+    /// assert_eq!(format!("{}", array), "[1, 2, 3, 4]");
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "[")?;
+        for (count, v) in self.iter_shared().enumerate() {
+            if count != 0 {
+                write!(f, ", ")?;
+            }
+            write!(f, "{}", v)?;
```

Comment:
```suggestion
            write!(f, "{v}")?;
```

Reply:
Btw: is this ChatGPT code? 😁

Reply:
Not at all :joy:. Just followed the patterns of rust by example and the std library docs.

Reply:
Ah, then ChatGPT stole it from there 😅